Software/smart-charger/src/utils/api/users.js

```diff
@@ -0,0 +1,31 @@
+import apiPath from "./api";
+
+export const registerUser = async (user) => {
+  try {
+    const res = await fetch(`${apiPath}/api/register`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(user),
+    });
+
+    const data = await res.json();
+    return data;
+  } catch (e) {}
```

Comment:
Since you are not implementing any logic for catch part I don't see the reason to have it here.

Reply:
You're right, I left it by mistake. It is removed now.